eva/parser/eva.lark

```diff
@@ -70,7 +70,7 @@ drop_udf: DROP UDF if_exists? udf_name
 
 //    Primary DML Statements
 
-delete_statement: DELETE FROM table_name (WHERE expression)? order_by_clause? (LIMIT decimal_literal)?
+delete_statement: DELETE FROM table_name (WHERE where_expr)? order_by_clause? (LIMIT decimal_literal)?
```

Comment:
Remove `order_by_clause`, `limit`, and `WHERE` can be optional. If the user omits the `where` clause, delete all the rows in the table. We can make omit this corner case if it requires significant changes.  

https://docs.snowflake.com/en/sql-reference/sql/delete.html
https://www.postgresql.org/docs/current/sql-delete.html